sklearn/linear_model/_cd_fast.pyx

```diff
@@ -476,7 +476,7 @@ def sparse_enet_coordinate_descent(floating [::1] w,
 
 
 def enet_coordinate_descent_gram(floating[::1] w,
-                                 floating alpha, floating beta,
+                                 floating l1_reg, floating l2_reg,
```

Comment:
Although I like this name change, I do not think we need it anymore for fixing the warning. We can have this refactor in another PR.